src/views/Main/index.jsx

```diff
@@ -269,12 +274,13 @@ class MainContainer extends Component {
                 )}
                 <PropsRoute
                   path="/teams"
-                  component={BugzillaComponents}
+                  component={Teams}
                   bugzillaComponents={Object.values(teamComponents)}
                   onComponentDetails={this.handleShowComponentDetails}
                 />
               </Switch>
             </Suspense>
+            {!doneLoading && <Spinner loading /> }
```

Comment:
Instead of adding `doneLoading`, could we instead just check if the context is loaded directly by inspecting the content of the variable?

Reply:
The code started getting tricky since we have to keep track of either `teamComponents` and `bugzillaComponents` (do a search for those strings on the file and you will see). For now I want to land it as-is and follow up if you would like me to improve this.

Reply:
@helfi92 are you fine if I go ahead as is?

Reply:
Go ahead.